apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -0,0 +1,340 @@
+package gov.cms.bfd.server.war.r4.providers;
```

Comment:
I would consider naming this file simply BeneficiaryTransformer.java - as it already resides in the r4 folder.

Reply:
Based on BFD v2 RFC, we have decided to have FHIR/HAPI version specific Resource providers and Transformers.